### ISSUE TITLE
Updating to VSCode 1.72.0 breaks autocomplete

### ISSUE THREAD

**Issue by ryanscherler** [withfig/fig#1883]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and tested again (tell us if that fixed it)

### Description:

It seems all of a sudden Fig stopped working in the integrated terminal of VSCODE. Is this possibly related to a recent update to VSCODE itself?

### Environment

```yaml
- SHELL: /bin/zsh
    - TERM: xterm-256color
    - TERM_SESSION_ID: b93b3e1a-ff1d-49f7-847e-1ab5f78f21c5
    - FIG_PARENT: b93b3e1a-ff1d-49f7-847e-1ab5f78f21c5
    - FIG_SET_PARENT: b93b3e1a-ff1d-49f7-847e-1ab5f78f21c5
- integrations:
  - SSH: false
  - TMUX: false
  - iTerm: application is not present.
  - Hyper: application is not present.
  - Visual Studio Code: installed!
  - Docker: false

* M1 Mac
```

**Comment by paulhmorris** [comment]
After updating to the latest version of VSCode (1.72.0 Universal), Fig has stopped working in the integrated terminal. It does appear to be related to the latest VSCode update, as downgrading to VSCode 1.71.2 brings back autocomplete.

**Comment by SteveSultan** [comment]
Same issue here after upgrade VSCode (1.72.0 Universal).

**Comment by mschrage** [comment]
Hey @paulhmorris! Thanks for reporting. We'll investigate and try to figure out what's going on.

**Comment by hopkins385** [comment]
Same here. 
VS Code Version: 1.72.0
fig_cli 1.5.0
Iterm2 Build 3.4.16

**Comment by mschrage** [comment]
Seems like VSCode has a regression in their Accessibility support. The application no longer responds to the [Electron convention](https://www.electronjs.org/docs/latest/tutorial/accessibility#macos) for enabling screen reader mode, which Fig requires in order to determine the cursor position.

I'll push a workaround out on the beta channel shortly.

**Comment by grant0417** [comment]
The fix for this is now live on beta (`1.0.62 B540`) and should be on stable in a day or two, let us know if this works for you!

**Comment by ryanscherler** [comment]
I saw Fig just auto-updated and Bam! it works! Thanks so much @grant0417  Y'all are fast!
